frontend/src/components/DataGrid.tsx

```diff
@@ -40,7 +40,6 @@ const defaultColDef: ColDef = {
   resizable: true,
   editable: false,
   headerComponentParams: createCustomHeader(lockIcon),
-  valueFormatter: (params) => (params.value === "null" ? "" : params.value),
```

Comment:
This code hides any cell values that return a "null" string.

I've confirmed that none of the existing fields show "null" string values on the dashboard.

We don't want to hide this because:
1. When this issue occurs, this line of code doesn't prevent "null" string values from showing up in the downloaded file.
2. We can more quickly catch and resolve this error at the query-level.